nursery/encrypt-data-using-rc4-via-systemfunction032.yml

```diff
@@ -0,0 +1,22 @@
+rule:
+  meta:
+    name: encrypt data using RC4 via SystemFunction032
+    namespace: data-manipulation/encryption/rc4
+    authors:
+      - richard.weiss@mandiant.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
+      - Cryptography::Encrypt Data::RC4 [C0027.009]
+    references:
+      - https://doxygen.reactos.org/df/d13/sysfunc_8c_source.html
+      - https://blog.gentilkiwi.com/tag/systemfunction032
+  features:
+    - or:
```

Comment:
```suggestion
    - and:
```

Reply:
I decided not to use AND and optional because there will be no capa api finding, if GetProcAddess is used.

Reply:
`optional` can also be written as `0 or more` which means it always evaluates to `true` so it shouldn't be used under an `or`. I'd recommend that it only be used under `and` to provide more context to the primary logic. 

I suspect that using `or` with `optional` would match in every possible scope, which is probably not wanted.